plugin-tester-scala/src/test/scala/example/myapp/helloworld/GreeterServiceSpec.scala

```diff
@@ -31,7 +31,7 @@ import scala.concurrent.{ Await, ExecutionContext }
 import scala.concurrent.duration._
 
 @RunWith(classOf[JUnitRunner])
-class GreeterSpec extends Matchers with AnyWordSpecLike with BeforeAndAfterAll with ScalaFutures {
```

Comment:
the fileName is GreeterServiceSpec in scala module. And in java moudle, it's JGreeterServiceSpec. So use the name